src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,114 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private List<Node<K, V>>[] table;
```

Comment:
Why do you use List[]. It should be Node<>[]

Reply:
> Why do you use List[]. It should be Node<>[]

This is easier than using the next pointer.

Reply:
It is easier but I think it uses more memory as just to create List object you need to use extra memory. But I sent this message yo mentors and will wait for result. And when you done correcting mark it as a - again stutend review

Reply:
https://prnt.sc/11bk23w

Reply:
> https://prnt.sc/11bk23w

fixed